components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/Fields.java

```diff
@@ -91,4 +93,11 @@ private static String formatDuration(Duration duration) {
 
         return s.toString().trim();
     }
+
+    private static String formatBigDecimal(BigDecimal value) {
+        if (value.compareTo(BigDecimal.ZERO) == 0) {
+            return "";
```

Comment:
-1: Why don't we return '0' for a zero serialization factor?  I would think that would be meaningful, as it would mean that the project isn't parallelized at all, correct?

Reply:
Since serialization factor is the ratio between cumulative task execution time and effective task execution time I don't believe it's possible for it to be `0.0`. A serialization factor of `1.0` would indicate that a project isn't parallelized at all. 

I did clean this up in [a subsequent PR](https://github.com/gradle/gradle-enterprise-build-validation-scripts/pull/283/files#diff-fae7ccc17156c0e5c419125e10074914f502fe4e12fdcdf6136603cac44950d4R78-R80).